addons/skin.estuary/xml/Variables.xml

```diff
@@ -5,6 +5,9 @@
 		<value condition="ListItem.HasTimer | ListItem.HasTimerSchedule">windows/pvr/timer.png</value>
 		<value condition="ListItem.HasArchive">windows/pvr/archive.png</value>
 	</variable>
+	<variable name="AutoCompletionContentVar">
+		<value condition="System.HasAddon(plugin.program.autocompletion) + !System.HasHiddenInput">plugin://plugin.program.autocompletion?info=autocomplete&amp;&amp;id=$INFO[Control.GetLabel(312).index(1)]&amp;&amp;limit=9</value>
```

Comment:
Correct me if I'm wrong, but shouldn't we use System.AddonIsEnabled now?

Reply:
you're absolutely right, thx!